File: algorithms/Codility-master/Codility-master/4_CountingElements/PermCheck.js


// 100%

function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    
    if (A.length == 0){
        return 0;
    }
    
    var s = A.sort(function(a, b){return a - b});
    for (let i =0 ; i< A.length ; i++){
        if ((s[i]-1 - i) != 0){
            return 0;
        }
    }
    return 1;
}

File: algorithms/Codility-master/Codility-master/4_CountingElements/PermCheck.py


# 100%
import collections
def solution(A):
    # write your code in Python 3.6
    if len(A) == 0:
        return 0
        
    s = sorted(A)
    for i in range(len(A)):
        if (s[i]-1 - i) != 0:
            return 0
            
    return 1